src/ReactEntity.js

```diff
@@ -36,7 +43,7 @@ class ReactEntity {
   }
 
   applyEntityConstructor(field, data) {
-     if (!data) return;
+     if (data === undefined || data === null) return;
```

Comment:
Don't think that `undefined` or `null` should be returned in a Boolean type. Any *falsy* values should default to `false`